Description: fix manpage-has-errors-from-man usr/share/man/man1/jhead.1.gz 54: warning: macro `&'' not defined
Author: Ludovic Rousseau <rousseau@debian.org>
--- a/jhead.1
+++ b/jhead.1
@@ -50,8 +50,8 @@
 .B jhead \-te "originals/&i" *.jpg
 
 would transfer the exif header for each .jpg file in the originals
-directory by the same name, Both Win32 and most Unix shells treat the
-'&' character in a special way, so you have to put quotes around that
+directory by the same name, Both Win32 and most Unix shells treat the '&'
+character in a special way, so you have to put quotes around that
 command line option for the '&' to even be passed to the program.
 
 .TP
@@ -279,8 +279,8 @@
 would create a thumbnail for each .jpg file in the thumbnails directory
 by the same name, (provided that the thumbnails directory exists, of
 course).  Both Win32 and UNIX shells treat the '&'character in a special
-way, so you have to put quotes around that command line option for the
-'&' to even be passed to the program.
+way, so you have to put quotes around that command line option for the '&'
+to even be passed to the program.
 
 If a '\-' is specified for the output file, the thumbnail is sent to
 stdout. (UNIX build only)
